api/v1beta1/hyperconverged_types.go

```diff
@@ -858,6 +864,23 @@ type HigherWorkloadDensityConfiguration struct {
 	MemoryOvercommitPercentage int `json:"memoryOvercommitPercentage,omitempty"`
 }
 
+// KubeMacPoolConfig defines kubemacpool MAC address range configuration
+// +k8s:openapi-gen=true
+type KubeMacPoolConfig struct {
+	// RangeStart defines the first MAC address in the kubemacpool range.
+	// The MAC address format should be AA:BB:CC:DD:EE:FF.
+	// This field is immutable once set.
+	// +optional
+	// +kubebuilder:validation:Pattern=`^[0-9a-fA-F]{2}:[0-9a-fA-F]{2}:[0-9a-fA-F]{2}:[0-9a-fA-F]{2}:[0-9a-fA-F]{2}:[0-9a-fA-F]{2}$`
```

Comment:
you can use this shorter regex pattern instead:
```
^([0-9A-Fa-f]{2}:){5}[0-9A-Fa-f]{2}$
```

Reply:
fixed, thanks!